components/items/ItemsGrid/utils/useSearchParams.ts

```diff
@@ -66,6 +66,25 @@ function useSearchByCollectionId() {
   }
 }
 
+function useSearchByCollections() {
+  const route = useRoute()
+
+  return {
+    collections: computed(() => {
+      if (route.query.collections) {
+        return [
+          {
+            collection: {
+              id_in: (route.query.collections as string).split(','),
```

Comment:
you have this kind of string manipulation bit spread all over each time you need to read/write collection from the url

please create a  util function and avoid repetition